app/src/main/java/com/poupa/vinylmusicplayer/adapter/artist/ArtistAdapter.java

```diff
@@ -98,7 +100,11 @@ public void onBindViewHolder(@NonNull final ViewHolder holder, int position) {
             }
         } else {
             if (holder.shortSeparator != null) {
-                holder.shortSeparator.setVisibility(View.VISIBLE);
+                if (PreferenceUtil.getInstance().useMaterialDesign()) {
```

Comment:
Just wondering if it's possible to create a utilty class `ImageTheme` that wraps all this if/else branches and only leave here something like:
```
holder.shortSeparator.setVisibility(ImageTheme.getAdapterSeparatorVisibility());
```

Reply:
This class can possibly contain as well the getItemListResource method you've proposed.

Reply:
My thoughts exactly :)